eng/pipelines/templates/stages/verify-installers.yml

```diff
@@ -54,9 +54,9 @@ parameters:
           ./test-telemetry-functions.sh "telemetry/macos11.sh.telemetry.csv" &&
           ./test-sh-install-errors.sh "bash" "$BASEURL" ""
 
-    Mac12Sh:
+    MacSh:
```

Comment:
Noticed the one above (`MacSh11`) is also using `MACVMIMAGE` - should one of these be using `MACVMIMAGE13` like with the `Pwsh` jobs?

Reply:
True. Good catch. I'll set it up as an N-1 just like the one below with pwsh.